live-examples/js-examples/regexp/regexp-prototype-dotall.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">
+var regex1 = new RegExp('foo', 's');
+
+console.log(regex1.dotAll);
+// expected output: true
+
+var regex2 = new RegExp('bar', 'i');
```

Comment:
Also use `const` here. And I'm not sure why you are passing the `'i'` flag? It would be clearer I think if you omitted it, unless there's something I am missing here.

Reply:
Hi, I used the 'i' flag to demonstrate the case of this function returning an output of 'false', if that makes sense.

Reply:
But I don't think you need `'i'` for this example, do you? `dotAll` will be `false` if you just don't pass any flags.

Reply:
I just checked and I see that it results in false without the flag as well. Should I update the example with no flag argument?

Reply:
Yes, please! I think it would be clearer like that.